### PR TITLE
Ben's First Attempt

### PR DESCRIPTION
**Pull request by benjaminpearson**
It takes 26 seconds to do 1000000000 on my Macbook Air.

### CODE REVIEW

File: challenges/odd-divisor/test/index.test.js [hunk 1]
```diff
@@ -11,7 +11,16 @@ const tests = [
...
+  [13, 63],
+  [14, 70],
+  [15, 85],
+  [16, 86],
+  [17, 103],
+  [18, 112],
+  [100, 3344],
+  [200, 13344],
+  [1000, 333396],
+  [1000000000, 333333333334181300], // @TODO: Confirm if correct value
```
Comment (jaypeng2015) [inline challenges/odd-divisor/test/index.test.js:23]: There is a better way to test.
Try `sum(1000000000-1)-sum(1000000000-2)`, what's your result?

File: challenges/odd-divisor/solutions/bens-first-attempt.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Ben's First Attempt
+ * Author: Ben Pearson
+ */
+
+function isEven(input) {
+  return input % 2 === 0;
+}
+
+function halfUntilOdd(input) {
+  const half = input / 2;
```
Comment (paulbao) [inline challenges/odd-divisor/solutions/bens-first-attempt.js:11]: May be try `const half = input * 0.5` would be a little bit faster?
https://stackoverflow.com/questions/226465/should-i-use-multiplication-or-division

Comment (jaypeng2015) [reply challenges/odd-divisor/solutions/bens-first-attempt.js:11]: I won't change much I think.

File: challenges/odd-divisor/test/index.test.js (hunk 1 above)
Comment (jaypeng2015) [reply challenges/odd-divisor/test/index.test.js:23]: Also I am pretty confident that the result should be `333333333334181226`

**Pull request closed without merging**